.eslint-doc-generatorrc.js

```diff
@@ -0,0 +1,13 @@
+"use strict"
+const config = {
+    urlConfigs: "https://github.com/eslint-community/eslint-plugin-n#-configs",
+    configEmoji: [
+        ["recommended-script", "✅"],
+        ["recommended-module", "🟢"],
+        ["recommended", "☑️"],
+        ["flat/recommended-script", "✅"],
+        ["flat/recommended-module", "🟢"],
```

Comment:
@bmish I am uncertain about the optimal approach here - having the same rules in both flat config and eslintrc config. Therefore, it may not be necessary to duplicate emojis. thoughts?

Reply:
To make sure I have the right context, is exporting a config twice (under one name for eslintrc, and under another name for flat config) the recommended approach to support both kinds of configs? Does ESLint mention a recommendation about this somewhere?

Using the same emoji for multiple configs, even if they are intended to represent a single config, seems confusing and breaks assumptions that each config has a unique emoji.

Some options to handle this:
1. eslint-doc-generator should not allow the same emoji for multiple configs. Perhaps we can recommend using the same emoji but different colors for eslintrc vs. flat?
2. eslint-doc-generator allows the same emoji for multiple configs, but better display/handling (e.g. avoid repeating the emoji in the readme rules table).
3. eslint-doc-generator adds some special handling/representation to indicate that one config is the flat version of an eslintrc config. But not sure how this can be reliably detected.

I personally prefer 1 as of now.

Reply:
I prefer 2>1>3.

for eslint plugins, there is not too much special eslintrc configs vs. flat configs. we just need to extend the `eslint-plugin-jsdoc` configs.